warehouse/templates/pages/help.html

```diff
@@ -152,6 +154,17 @@ <h2>{{ project_name_claim() }}</h2>
         </p>
       </section>
 
+      <section id="file_size_limit" class="common-question">
```

Comment:
I think our frontend generally prefers `kebab-case` for ids rather than `snake_case`.

Reply:
Got it. Fixing.